src/electronApp/ElectronEnvironmentAdapter.ts

```diff
@@ -1,30 +1,31 @@
-import { exec, ChildProcess } from 'child_process'
-import { EnvironmentAdapter } from '../core/environmentAdapter'
-import { log } from '../core/logService'
-import { shell } from 'electron'
+import {exec, ChildProcess} from 'child_process'
+import {EnvironmentAdapter} from '../core/environmentAdapter'
+import {log} from '../core/logService'
+import {shell} from 'electron'
 
 export class ElectronEnvironmentAdapter implements EnvironmentAdapter {
 
-	public getEnvironmentName(): 'electron' {
-		return 'electron'
-	}
+    public getEnvironmentName(): 'electron' {
+        return 'electron'
+    }
 
-	public runShellCommand(command: string): ChildProcess {
-		log.info(`executing '${command}'`)
-		return exec(command, (error: unknown|null, stdout: string, stderr: string) => {
-			if (error) {
-				log.warning(error.toString())
-			}
-		})
-	}
+    public runShellCommand(command: string, options: { cwd: string }): ChildProcess {
```

Comment:
do we want to require options always or should they be optional?
```suggestion
    public runShellCommand(command: string, options?: { cwd?: string }): ChildProcess {
```